docs/tempo/website/operations/deployment.md

```diff
@@ -28,3 +28,8 @@ Helm charts are available in the grafana/helm-charts repo:
 
 - [single binary](https://github.com/grafana/helm-charts/tree/main/charts/tempo)
 - [microservices](https://github.com/grafana/helm-charts/tree/main/charts/tempo-distributed)
+
+## Kubernetes manifests
+
+You can find example Kubernetes manifests in [operations/kube-manifests](https://github.com/grafana/tempo/tree/main/operations/kube-manifests).
```

Comment:
```suggestion
You can find examples of Kubernetes manifests in [operations/kube-manifests](https://github.com/grafana/tempo/tree/main/operations/kube-manifests) folder.
```

Reply:
[kube-manifests](https://github.com/grafana/tempo/tree/main/operations/kube-manifests) doesn't contain multiple examples, it contains a bunch of files which you are one example together. "examples of Kubernetes manifests" sounds like we have multiple examples.

How about this?

```suggestion
You can find a collection of Kubernetes manifests to deploy Tempo in [operations/kube-manifests](https://github.com/grafana/tempo/tree/main/operations/kube-manifests) folder.
```

What do you think?

Reply:
Awesome! Just one suggestion:

You can find a collection of Kubernetes manifests to deploy Tempo in the [operations/kube-manifests](https://github.com/grafana/tempo/tree/main/operations/kube-manifests) folder.